src/aleph_client/models.py

```diff
@@ -102,3 +102,56 @@ def from_unsanitized_input(
             url=url,
             hash=hash,
         )
+
+
+class CRNInfo(BaseModel):
+    hash: ItemHash
+    name: str
+    url: str
+    version: Optional[str]
+    score: float
+    stream_reward_address: str
+    machine_usage: Optional[MachineUsage]
+    qemu_support: Optional[bool]
+    confidential_computing: Optional[bool]
+
+    @property
+    def display_cpu(self) -> str:
+        try:
```

Comment:
Don't abuse the `try`, it hide errors (and Rich as is already hide plenty errors)